docs/command/atlas-clusters-update.txt

```diff
@@ -18,6 +18,8 @@ You can specify modifications in a JSON configuration file with the --file flag.
 		
 You can't change the name of the cluster or downgrade the MongoDB version of your cluster.
 
+You must not update a replica set to a multi-sharded cluster, only single-sharded cluster is supported. See https://www.mongodb.com/docs/atlas/scale-cluster/#convert-a-replica-set-to-a-sharded-cluster and https://www.mongodb.com/docs/upcoming/tutorial/convert-replica-set-to-replicated-shard-cluster for more information on this type of update.
```

Comment:
@mongodb/docs-cloud-team If we have the shortened/permanent links, could you please share so that i can replace them here? thanks!

Reply:
oh ok! I thought that w/ the dochub ones we wouldn't have the URL change problem, if that's the case then I'll resolve the comment, thanks!